src/UIComponents/SaveModel.jsx

```diff
@@ -22,21 +22,26 @@ class SaveModel extends Component {
   getFields = () => {
     var fields = [];
 
-    fields.push({ id: "name", text: "Name" });
+    fields.push({ id: "name", text: "What will you name the model?" });
 
     for (const columnDescription of this.props.columnDescriptions) {
       fields.push({
         id: columnDescription.id,
         isColumn: true,
-        text: "Describe " + columnDescription.id,
+        text: "Description for: " + columnDescription.id,
         answer: columnDescription.description
       });
     }
 
-    fields.push({ id: "potentialUses", text: "How can this model be used?" });
+    fields.push({
+      id: "potentialUses",
+      text: "How can this model be used?",
+      placeholder: "Write a brief description"
```

Comment:
teeny tiny nit: do we want punctuation at the end of the placeholder?

Reply:
Good catch! I added a period and updated the screenshots :)